paket.lock

```diff
@@ -429,6 +520,7 @@ NUGET
       System.Runtime (>= 4.3)
     System.Diagnostics.DiagnosticSource (6.0)
       System.Runtime.CompilerServices.Unsafe (>= 6.0)
+    System.Diagnostics.EventLog (7.0)
```

Comment:
Should we prevent this from being 7.0 by adding strategy: min to paket.dependencies?